automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/Modals.scala

```diff
@@ -65,3 +65,14 @@ case class MessageModal(implicit webDriver: WebDriver) extends OKCancelModal {
   }
 }
 
+case class RequestAccessModal(implicit webDriver: WebDriver) extends OKCancelModal {
+  val requestAccessText = "For access to TCGA controlled data please apply for access via dbGaP"
+  def validateLocation: Boolean = {
+    testId("push-message").element != null
+  }
+
+  def getRequestAccessText: String = {
```

Comment:
"get" is unnecessary. In fact, IntelliJ warns about functions beginning with "get" in certain circumstances. Instead, this could be something like `readRequestAccessText`... or even simply `messageText`; the class name `RequestAccessModal` gives context to what the text is for.

Reply:
This modal is specific to the Library page so we should handle this in DataLibraryPage

Reply:
Oh, and the one I was working with recently was on the workspace list page. Wow... too bad these are different/inconsistent. That's why I was confused about ok/cancel below.

Reply:
Hmm.. maybe we should standardize modal verification? Either:
modals should use the same data-test-id for all messages and we verify the text
OR the data-test-id is unique for the message and we verify that the data-test-id is present.